src/index.js

```diff
@@ -4,50 +4,67 @@ const github = require('@actions/github')
 const { logInfo } = require('./log')
 const { getInputs } = require('./util')
 
-const { GITHUB_TOKEN, MERGE_METHOD, EXCLUDE_PKGS, MERGE_COMMENT } = getInputs()
+const {
+  GITHUB_TOKEN,
+  MERGE_METHOD,
+  EXCLUDE_PKGS,
+  MERGE_COMMENT,
+  APPROVE_ONLY,
+} = getInputs()
 
-async function run () {
+async function run() {
   try {
     const octokit = github.getOctokit(GITHUB_TOKEN)
 
     const { repository, pull_request: pr } = github.context.payload
+
+    if (!pr) {
+      throw new Error(
```

Comment:
Shouldn't we silently ignore?

Reply:
we could return. I kept the existing semantics though, just breaking it in a more explicit way. without this it would still break here https://github.com/fastify/github-action-merge-dependabot/pull/9/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R29